src/Console/Command/StageCommand.php

```diff
@@ -3,21 +3,72 @@
 namespace PhpTuf\ComposerStager\Console\Command;
 
 use PhpTuf\ComposerStager\Console\Misc\ExitCode;
+use PhpTuf\ComposerStager\Domain\Stager;
 use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class StageCommand extends Command
 {
     protected static $defaultName = 'stage';
 
+    /**
+     * @var \PhpTuf\ComposerStager\Domain\Stager
+     */
+    private $stager;
+
+    public function __construct(Stager $stager)
+    {
+        parent::__construct(static::$defaultName);
```

Comment:
Is there any need to pass `static::$defaultName` here? Not harmful, just seems a bit strange.